flow/genericactors.actor.h

```diff
@@ -707,7 +707,7 @@ class AsyncTrigger : NonCopyable {
 // Binds an AsyncTrigger object to an AsyncVar, so when the AsyncVar changes
 // the AsyncTrigger is triggered.
 ACTOR template <class T>
-void forward(Reference<AsyncVar<T> const> from, AsyncTrigger* to) {
+Future<Void> forward(Reference<AsyncVar<T> const> from, AsyncTrigger* to) {
```

Comment:
Were you able to confirm whether there were any other callers of this function that would be affected by this change?

Reply:
Ah, I see the comment on the other review.